### PR TITLE
Add helm test action

### PR DESCRIPTION
**Pull request by liam-verta**
After #1 is merged, we can try this to ensure future PRs have at least basic chart testing.
Copied from https://github.com/helm/chart-testing-action#example-workflow 
Signed-off-by: Liam Newman <96086065+liam-verta@users.noreply.github.com>

### CODE REVIEW

File: .github/workflows/helm-test.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (Pokom) [inline .github/workflows/helm-test.yml:1]: NIT: Can probably remove this helper comment

File: .github/workflows/helm-test.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --target-branch ${{ github.event.repository.default_branch }}
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.4.0
+        if: steps.list-changed.outputs.changed == 'true'
```
Comment (Pokom) [inline .github/workflows/helm-test.yml:51]: I haven't seen this before, so please disregard if irrelevant: Will this run anything without a `run` command passed to it?

**Review by Pokom — APPROVED** [review]
Just two minor comments, otherwise :+1:

File: .github/workflows/helm-test.yml (hunk 1 above)
Comment (liam-verta) [reply .github/workflows/helm-test.yml:1]:
```suggestion
```

File: .github/workflows/helm-test.yml (hunk 2 above)
Comment (liam-verta) [reply .github/workflows/helm-test.yml:50]: Don't know.

Comment (liam-verta) [reply .github/workflows/helm-test.yml:50]: Ah, which is see has a bug in it. https://github.com/liam-verta/opencost-helm-chart/actions/runs/4000497937

File: .github/workflows/helm-test.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  helm-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (liam-verta) [inline .github/workflows/helm-test.yml:21]:
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/helm-test.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.3.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
```
Comment (liam-verta) [inline .github/workflows/helm-test.yml:42]:
```suggestion
            echo "changed=true" >> $GITHUB_OUTPUT
```

File: .github/workflows/helm-test.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (liam-verta) [inline .github/workflows/helm-test.yml:1]:
```suggestion
# CI Workflow for helm chart
```

**Comment by liam-verta** [comment]
Okay, I finally got a version that passes lint and install.

File: Chart.yaml [hunk 6]
```diff
@@ -2,4 +2,7 @@ apiVersion: v2
 appVersion: 1.9.8
 name: opencost
 description: Opencost and Opencost UI
-version:  0.0.1
+version: 0.0.1
```
Comment (liam-verta) [inline Chart.yaml:5]: Linting in `ct` is really strict by default. I'm okay with that.

**Comment by liam-verta** [comment]
https://github.com/liam-verta/opencost-helm-chart/actions/runs/4001425634/jobs/6867635644 - success. 

This change in mergeable.

File: Chart.yaml [hunk 7]
```diff
@@ -2,4 +2,7 @@ apiVersion: v2
...
+maintainers:
+  - name: mattray
```
Comment (mattray) [inline Chart.yaml:7]: I guess this is OK for now, I'd prefer to get some more names or a catch-all account

**Pull request merged**